stock_request_tier_validation/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2019-2020 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Request Tier Validation",
+    "summary": "Extends the functionality of Stock Requests to "
+    "support a tier validation process.",
+    "version": "13.0.1.0.0",
+    "category": "Warehouse Management",
```

Comment:
Change this to 'Warehouse' as it won't appear in good category on OCA shop

Reply:
Changed.